### PR TITLE
REPO-4471 Update all-amps docker image for 6.0.N

### PR DESCRIPTION
**Pull request by andrei-zapodeanu-alfresco**

### CODE REVIEW

**Comment by cristianturlica** [comment]
Changes look good to me, only thing to consider is the s3 properties should be removed and loaded from system variables.

**Comment by andrei-zapodeanu-alfresco** [comment]
The removal of properties will be done in a different commit since it needs to be done in other branches at well.
This way, it will be easier to just cherry pick that commit across all branches.

File: docker-alfresco/Dockerfile [hunk 1]
```diff
@@ -91,8 +91,8 @@ RUN mkdir -p ${TOMCAT_DIR}/conf/Catalina/localhost && \
     chmod g+rwx ${TOMCAT_DIR}/work && \
     chmod o-w ${TOMCAT_DIR}/work
 
-# To remote debug into this image add: EXPOSE 8000
-# Changes are also required to the docker-compose/docker-compose.yml file.
+# To remote debug into this image add: EXPOSE 8000	EXPOSE 10001
```
Comment (alandavis) [inline docker-alfresco/Dockerfile]: ???

Comment (andrei-zapodeanu-alfresco) [reply docker-alfresco/Dockerfile]: Removed

**Pull request merged**